### PR TITLE
feat: Concept of the help for keyboard shortcuts TG-297

### PR DESCRIPTION
**Pull request by stepan662**

### CODE REVIEW

File: webapp/src/views/projects/translations/ShortcutsHint.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  },
+  hoverHidden: {
+    opacity: 0,
+    transition: 'opacity 300ms ease-in-out',
+  },
+}));
+
+type Shortcut = {
+  name: React.ReactNode;
+  meta?: string;
```
Comment (JanCizmar) [inline webapp/src/views/projects/translations/ShortcutsHint.tsx:81]: Maybe we can be more specific here like meta?: 'CTRL' | 'CMD'

File: webapp/src/views/projects/translations/TranslationOpened.tsx [hunk 2]
```diff
@@ -166,6 +179,24 @@ export const TranslationOpened: React.FC<Props> = ({
               onCancel={onCancel}
               onStateChange={onStateChange}
             />
+            <ShortcutsHint
+              items={[
+                {
+                  name: <T>{translationStates[nextState].translationKey}</T>,
```
Comment (JanCizmar) [inline webapp/src/views/projects/translations/TranslationOpened.tsx:185]: `TypeError: Cannot read properties of undefined (reading 'translationKey')` is thrown when I try to edit untranslated value. Please add test for this.

**Review by JanCizmar — CHANGES_REQUESTED** [review]
It looks nice, but It doesn't make the process much faster. 

I am still not able to review the translations with hitting single shortcut without moving between translations with mouse or hitting tab many times. I think, that combination of moving with arrows and allowing users to to focus the key items like key or specific translations without opening editor would be the best option. In the focused item or with hover (which can do both the same thing, actually) we can show the shortcuts the way Zuzka designed.

File: e2e/cypress/common/shortcuts.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import { waitForGlobalLoading } from './loading';
+
+export const IS_MAC = Boolean(navigator.userAgent.includes('Mac OS X'));
```
Comment (JanCizmar) [inline e2e/cypress/common/shortcuts.ts:3]: What about older Macs? ❓ 🥇

Comment (stepan662) [reply e2e/cypress/common/shortcuts.ts]: Ok, switched to Mac

**Pull request merged**